lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.rb

```diff
@@ -9,6 +9,10 @@ def initialize
     @template = '179d-90.1-2007'
     @whole_building_space_type_name = 'WholeBuilding'
     load_standards_database
+
+    # This is super weird, but this is for resolving ventilation and exhaust
+    # per the space type's... and merging with the rest
+    @std_2007 = ASHRAE9012007.new
```

Comment:
So your Class derives ASHRAE9012007, and an instance of our class also carries an instance of the parent ASHRAE9012007 class...